custom_rules/rules/command_and_control_dac_demo_dev_rule_2.toml

```
@@ -16,7 +16,7 @@ interval = "5m"
 language = "eql"
 license = "Elastic License v2"
 max_signals = 100
-name = "DAC Demo Dev Rule 2 Updated"
+name = "DAC Demo Dev Rule 2 Feedback"
 note = "Test Note"
```

```suggestion
note = "This rule is part of a demo"
```
I think this update should also include an update to the note.